src/main/java/fr/insalyon/citi/golo/runtime/FunctionCallSupport.java

```diff
@@ -108,11 +118,15 @@ public static Object fallback(FunctionCallSite callSite, Object[] args) throws T
       Method method = (Method) result;
       checkLocalFunctionCallFromSameModuleAugmentation(method, callerClass.getName());
       types = method.getParameterTypes();
-      if (method.isVarArgs() && isLastArgumentAnArray(types.length, args)) {
+      //TODO improve varargs support on named arguments. Matching the last param type + according argument
```

Comment:
a function call w/ named args + varargs found actually a varargs method but this test also check that the last arg is an array... In a named args context, the varargs arry could be at any place in the function call.
I thought about flagging the varargs parameter in the `@GoloFunction` but don't know if it's worthless. 
So far all the tests run successfully so decided the _good enough impl._ (aka. Quick&Dirty :smile: )